examples/contour/01-contour-config.yaml

```diff
@@ -22,7 +22,7 @@ data:
     #   configmap-namespace: leader-elect
     ### Logging options
     # Default setting
-    # accesslog-format: clf
+    accesslog-format: clf
```

Comment:
Is this a valid setting?

Reply:
Yep, anything not 'json' will set the logging to the current logging format.